core/modules/util/MultiError.h

```diff
@@ -31,7 +31,7 @@
  */
 
 #ifndef LSST_QSERV_UTIL_ERROR_H
-#define LSST_QSERV_UTIL_ERRORCONTAINER_H
+#define LSST_QSERV_UTIL_ERROR_H
```

Comment:
Correct name for include guard should be `LSST_QSERV_UTIL_MULTIERROR_H`. Also check comment for `#endif`